concept/type/owns.feature

```diff
@@ -0,0 +1,3952 @@
+    When put attribute type: custom-attribute
+    When attribute(custom-attribute) set value-type: <value-type>
+    When entity(person) set owns: custom-attribute
+    Then entity(person) get owns(custom-attribute) set annotation: @unique; fails
+    Then entity(person) get owns(custom-attribute) get annotations is empty
+    When transaction commits
+    When connection open read transaction for database: typedb
+    Then entity(person) get owns(custom-attribute) get annotations is empty
+    Examples:
+      | value-type    |
+      | double        |
+      | decimal       |
+      | custom-struct |
+
+  # TODO: Change the test if owns can set @unique annotation for lists!
+  Scenario Outline: Owns cannot set @unique annotation for lists
+    When put attribute type: custom-attribute
+    When attribute(custom-attribute) set value-type: <value-type>
+    When entity(person) set owns: custom-attribute[]
+    Then entity(person) get owns(custom-attribute[]) set annotation: @unique; fails
+    Then entity(person) get owns(custom-attribute[]) get annotations is empty
+    When transaction commits
+    When connection open read transaction for database: typedb
+    Then entity(person) get owns(custom-attribute[]) get annotations is empty
+    Examples:
+      | value-type    |
+      | long          |
+      | string        |
+      | boolean       |
+      | double        |
+      | decimal       |
+      | datetime      |
+      | datetimetz    |
+      | duration      |
+      | custom-struct |
+
+########################
+# @values
+########################
+
+  Scenario Outline: Owns can set @values annotation for <value-type> value type and lists and unset it
+    When put attribute type: custom-attribute
+    When attribute(custom-attribute) set value-type: <value-type>
+    When put attribute type: custom-attribute-2
+    When attribute(custom-attribute-2) set value-type: <value-type>
+    When entity(person) set owns: custom-attribute
+    When entity(person) get owns(custom-attribute) set annotation: @values(<args>)
+    Then entity(person) get owns(custom-attribute) get annotations contain: @values(<args>)
+    When entity(person) set owns: custom-attribute-2[]
+    When entity(person) get owns(custom-attribute-2[]) set annotation: @values(<args>)
+    Then entity(person) get owns(custom-attribute-2[]) get annotations contain: @values(<args>)
+    When transaction commits
+    When connection open schema transaction for database: typedb
+    Then entity(person) get owns(custom-attribute) get annotations contain: @values(<args>)
+    Then entity(person) get owns(custom-attribute-2[]) get annotations contain: @values(<args>)
+    When entity(person) unset owns: custom-attribute
+    When entity(person) unset owns: custom-attribute-2[]
+    Then entity(person) get owns is empty
+    When transaction commits
+    When connection open read transaction for database: typedb
+    Then entity(person) get owns is empty
+    Examples:
```

Comment:
It can get better if I shorten some of the `long` and `datetimetz` cases, but they deserve to be here... I guess =)